scripts/DeployBridges.s.sol

```diff
@@ -60,3 +61,27 @@ contract DeployArbBridgeArbitrum is ArbitrumScript {
     new AaveArbEthERC20Bridge{salt: salt}(0x3765A685a401622C060E5D700D9ad89413363a91);
   }
 }
+
+contract DeployCcipGhoBridgeEthereum is EthereumScript {
+  function run() external broadcast {
+    bytes32 salt = 'GHO Chainlink CCIP Bridge';
+    new CcipGhoBridge{salt: salt}(
+      0x80226fc0Ee2b096224EeAc085Bb9a8cba1146f7D,
+      0x514910771AF9Ca656af840dff83E8264EcF986CA,
+      0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f,
+      0x3765A685a401622C060E5D700D9ad89413363a91
+    );
+  }
+}
+
+contract DeployCcipGhoBridgeArbitrum is ArbitrumScript {
+  function run() external broadcast {
+    bytes32 salt = 'GHO Chainlink CCIP Bridge';
+    new CcipGhoBridge{salt: salt}(
+      0x141fa059441E0ca23ce184B6A78bafD2A517DdE8,
```

Comment:
What are these addresses?

Reply:
Please add a name to them or at least a link to Etherscan and a brief description.